src/constants.ts

```diff
@@ -13,3 +13,17 @@ export const URLS = {
 } as const
 
 export type URLS = typeof URLS
+
+export enum ButtonSize {
+  LARGE = 'large',
+  SMALL = 'small',
+}
+export enum ButtonVariant {
+  DARK = 'dark',
+  LIGHT = 'light',
+  AUTO = 'auto',
+}
+export enum PaymentMethodType {
+  MERCHANT = 'merchant',
+  CUSTOMER = 'customer',
+}
```

Comment:
Please, replace enums with literal string union type. It will be more convenient for 3rd party integrations and doesn't requires additional imports (which should be documented).